src/analyses/apron/octApronPriv.apron.ml

```diff
@@ -674,12 +674,16 @@ end
 
 let priv_module: (module S) Lazy.t =
   lazy (
+    let check_mutex_enabled () =
+      let octapron_active = List.exists (fun x -> Json.string x="octApron") (GobConfig.get_list "ana.activated") in
```

Comment:
Maybe it makes sense to do this check in each `init` instead. That should only get called for activated analyses and would avoid this extra check when registering `octApron`.